libc-bottom-half/cloudlibc/src/libc/stdlib/qsort.c

```diff
@@ -1,15 +0,0 @@
-// Copyright (c) 2015 Nuxi, https://nuxi.nl/
```

Comment:
Why is not needed anymore?

Reply:
We use the "libc top half"'s `qsort, which is to say, musl's `qsort`.

Reply:
Gotcha